main_routers/system_router.py

```
@@ -733,145 +958,148 @@ async def proactive_chat(request: Request):
+        # {内心活动/Settings}...
+        # 现在时间...整理了近期发生的事情。
+        # Name | Content
+        # ...
+        
+        raw_memory_context = ""
         try:
             async with httpx.AsyncClient() as client:
                 resp = await client.get(f"http://127.0.0.1:{MEMORY_SERVER_PORT}/new_dialog/{lanlan_name}", timeout=5.0)
-                memory_context = resp.text
+                raw_memory_context = resp.text
         except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)